### DISCUSSION TITLE
App in Display

### DISCUSSION THREAD

**Discussion by RalphM77** [Thyraz/Sonos-Kids-Controller#9] [Q&A]
Dear all

Great Project.
I was able to rebuild the Kids Controller in just a few hours! Great.

But the small touch screen shows the Audio icons very small and just in the upper third of the screen. Below is a large black area.
The Raspi is using the full display...that is not the problem.
But the kiosk mode of chromium does not use the screen so the icons are too small for the kids.
Same behavior on a desktop monitor.
800X480 is already set...
Any ideas to help?!

best regards

Ralph

**Comment by Thyraz** [comment]
Hi Ralph,

the software is optimized for 800x480. 
So it's normal behavior, that it looks a little bit small on your desktop monitor.

Not sure what's exactly wrong on your Raspi box.
Can you post a screenshot?

Regards,
Tobias

**Reply by RalphM77** [reply]
Hey Tobias

Thanks for the fast reply.
After setting the resolution with the Raspi GUI it works now ... Strange.
With Shell it did not work.

Best regards

Ralph
